3rdparty/ErrorHandling/MavenException.cpp

```diff
@@ -9,7 +10,7 @@ MavenException::MavenException(ErrorCodes::Errors errCd, const std::string& deta
     message += details;
     message += "\n";
 
-    LOGD << message;
+    ElMavenLogger::getInstance()->logErr(message, ElMavenLogger::info);
```

Comment:
@rish9511 Why as `ElMavenLogger::info` and not something like critical or fatal?

Reply:
@sahil21 
I plan to implement these levels in the next version
Will add to TODO

Reply:
@rish9511 Thanks